README.md

```diff
@@ -1,8 +1,10 @@
 ![test-badge](https://github.com/SBC-Utrecht/pytom-match-pick/actions/workflows/unit-tests.yml/badge.svg?branch=main)
-# GPU template matching for cryo-ET
+# pytom-match-pick: GPU template matching for cryo-ET
 
 GPU template matching, originally developed in [PyTom](https://github.com/SBC-Utrecht/PyTom), as a standalone python package that can be run from the command line.
 
+![cover_image](images/tomo200528_100_illustration.png)
```

Comment:
This does not look OK in the rich diff, but does look ok when I checkout the branch on github

Reply:
for me it looks fine in the rich diff, what looks off for you?

Reply:
It looked like a regular link (like the Pytom one on the line above it)